core/src/main/scala/com/salesforce/op/ModelInsights.scala

```diff
@@ -527,10 +530,9 @@ case object ModelInsights {
     blacklistedMapKeys: Map[String, Set[String]],
     rawFeatureDistributions: Array[FeatureDistribution]
   ): Seq[FeatureInsights] = {
-    val contributions = getModelContributions(model)
-
     val featureInsights = (vectorInfo, summary) match {
       case (Some(v), Some(s)) =>
+        val contributions = getModelContributions(model, Option(v.getColumnHistory().length))
```

Comment:
did you mean `Option(v.getColumnHistory()).map(_.length)`? how do we even get null here?! and the same below.

Reply:
It doesnt look like you are actually using any of the additional info in the column history - you can just use v.columns.length